lib/portlayer/exec/container.go

```diff
@@ -377,15 +433,15 @@ func (c *Container) start(ctx context.Context) error {
 
 	detail, err = c.vm.WaitForKeyInExtraConfig(ctx, key)
```

Comment:
once we've powered the VM on, this should be StateUnknown on general error paths - this allows subsequent calls to attempt an update.